csharp-package/brainflow/brainflow/data_handler_library.cs

```diff
@@ -63,6 +63,8 @@ class DataHandlerLibrary64
         [DllImport ("DataHandler.dll", SetLastError = true, CallingConvention = CallingConvention.Cdecl)]
         public static extern int perform_wavelet_denoising (double[] data, int data_len, string wavelet, int decomposition_level);
         [DllImport ("DataHandler.dll", SetLastError = true, CallingConvention = CallingConvention.Cdecl)]
+        public static extern int perform_windowing (int window_function, int window_len, double[] window_data);
+        [DllImport ("DataHandler.dll", SetLastError = true, CallingConvention = CallingConvention.Cdecl)]
```

Comment:
in this file you need to do it for all OSes(there are ~3 more places where code like this needed